projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -143,7 +146,8 @@ export class TableWidgetRendererComponent
       pairwise(),
       filter(([previous, current]) => !isEqualIgnoreFunctions(previous, current)),
       map(([_, current]) => current),
-      share()
+      share(),
+      tap(() => this.onDashboardRefresh())
```

Comment:
Do we need this here or can we do it as part of onModeChange (which I'm guessing is why we now need it, to get new data when the mode changes too)

Reply:
Correct, putting it in onModeChange fixes the issue for this PR. I started with it here.

But there is an existing issue you can see in dev by starting from the Service list screen and refreshing. The list will be blank unless you hit the Refresh button at the top. It also works if you navigate in from another page. But adding the refresh here fixes the initial load issue.

Reply:
Not sure I follow, but will try to follow up on that separately if it's existing.